node/external/transactionAPI/gasUsedAndFeeProcessor_test.go

```diff
@@ -405,7 +342,7 @@ func TestComputeAndAttachGasUsedAndFeeFailedRelayedV1(t *testing.T) {
 	txWithSRefundSCR.GasUsed = 0
 
 	gasUsedAndFeeProc.computeAndAttachGasUsedAndFee(txWithSRefundSCR)
-	require.Equal(t, uint64(1274230), txWithSRefundSCR.GasUsed)
-	require.Equal(t, "1274230000000000", txWithSRefundSCR.Fee)
+	require.Equal(t, uint64(6148000), txWithSRefundSCR.GasUsed)
```

Comment:
why did these change? they are for relayed v1?

Reply:
they needed an update as I also deleted [this early check](https://github.com/multiversx/mx-chain-go/pull/6546/files#diff-94c96e7c2fd37481af33b7c4280ed747e24f9e59ce0863a3544bd64e1a351c53L77) that was misbehaving for v1

Reply:
ok